specs/capri/Monitoring Design.md

```diff
@@ -0,0 +1,734 @@
+
+# Telemetry Design
+
+
+**Authors:** [Xing Yang](<https://github.com/xing-yang>), [Sanil Kumar D](https://github.com/skdwriting), [Prakash R](https://github.com/rhsakarpos), [Najmudheen CT](https://github.com/NajmudheenCT)
+
+This documentation serves as the design spec for monitoring and alerting
+
+## Background
+------------
```

Comment:
This line ---- can be removed because ## already means this is the title for a subsection.